HISTORY.md

```diff
@@ -1,5 +1,9 @@
 Netuitive Linux Agent Release History
 ===============================
+Version 0.6.6
```

Comment:
Typically the version is only set directly on `dev` right before cutting over to `master` and tagging. That's so if additional changes come in which make it more than a bug fix the release doesn't have to be changed on a feature branch.

Reply:
agreed, but since we plan to merge to master and have it tested in UAT asap. so I kind of let this  one in the PR.